tests/runtime_tests.cpp

```diff
@@ -79,8 +81,17 @@ struct LambdaRuntimeTest : public ::testing::Test {
         create_function_request.SetFunctionName(function_name);
         // I ran into eventual-consistency issues when creating the role dynamically as part of the test.
         create_function_request.SetRole(get_role_arn("integration-tests"));
+
+        struct stat s;
+        auto rc = stat(ZIP_FILE_PATH, &s);
+        ASSERT_EQ(rc, 0) << std::string("file does not exist: ") + ZIP_FILE_PATH;
+        Aws::Utils::CryptoBuffer zip_file_bytes(s.st_size);
+        auto* zip_file = fopen(ZIP_FILE_PATH, "r");
+        fread(zip_file_bytes.GetUnderlyingData(), sizeof(unsigned char), s.st_size, zip_file);
+        fclose(zip_file);
+
         Model::FunctionCode funcode;
-        funcode.WithS3Bucket(S3BUCKET).WithS3Key(build_resource_name(S3KEY));
+        funcode.SetZipFile(zip_file_bytes);
```

Comment:
Move rather than copy since those are probably a few hundred kilobytes if not more. `funcode.SetZipFile(std::move(zip_file_bytes));`